internal/commands/result.go

```diff
@@ -1124,9 +1125,11 @@ func ReadResults(
 	}
 
 	if resultsModel != nil {
-		resultsModel, err = enrichScaResults(resultsWrapper, scan, params, resultsModel)
-		if err != nil {
-			return nil, err
+		if !isThresholdCheck {
```

Comment:
suggestions:
if resultsModel != nil && !isThresholdCheck {
.....
}

would be more readable

Reply:
there is a continue to this if that not related to SCA (line 1135)